src/tictactoe_brain/tictactoeBrain.cpp

```diff
@@ -208,25 +228,25 @@ int tictactoeBrain::victory_move()
     return -1;
 }
 
-bool tictactoeBrain::set_movement_type(bool b)
-{
-    movement_type=b;
-    baxter_tictactoe::SetTrajectoryType srv;
-    srv.request.smooth=(movement_type?true:false);
-
-    if(_clnt_movement_type.call(srv))
-    {
-        if (!srv.response.error)
-        {
-            ROS_INFO_STREAM("[tictactoeBrain] Movements set to " << (srv.request.smooth?"smooth":"mechanistic"));
-            return true;
-        }
-        ROS_ERROR_STREAM("[tictactoeBrain] Error setting movements to " << (srv.request.smooth?"smooth":"mechanistic"));
-        return false;
-    }
-    ROS_ERROR("[tictactoeBrain] Failed to call service set_movement_type");
-    return false;
-}
+// bool tictactoeBrain::set_movement_type(bool b)
```

Comment:
If you don't use this function at all, you can remove it and we will live happily ever after :)